### PR TITLE
[Kotlin Migration] CompatV23

### PR DESCRIPTION
**Pull request by Akshay0701**

### CODE REVIEW

File: AnkiDroid/src/main/java/com/ichi2/compat/CompatV23.kt [hunk 1]
```diff
@@ -14,39 +14,35 @@
...
-        picker.setMinute(minute);
+open class CompatV23 : CompatV21(), Compat {
+    override fun setTime(picker: TimePicker, hour: Int, minute: Int) {
+        picker.hour = hour
+        picker.minute = minute
     }
 
-    @Override
-    public int getHour(TimePicker picker) { return picker.getHour(); }
+    override fun getHour(picker: TimePicker): Int {
```
Comment (david-allison-1) [inline AnkiDroid/src/main/java/com/ichi2/compat/CompatV23.kt:33]: some of these were on one line, could this be kept?

Alt+Enter on the `{` and Android Studio will suggest the refactoring

**Review by david-allison-1 — APPROVED** [review]
LGTM, one optional nit

**Review by mikehardy — APPROVED** [review]
I don't mind the spacing so much - one line functions are a neat trick but nothing important and I'm passing through the PR queue right now, will merge as is if it passes through my local CI grinder (which it should...)

**Comment by mikehardy** [comment]
Hi there! FEBRUARY OpenCollective Notice

Just a friendly notice that we try to process OpenCollective payments monthly - it's time for [February 2022 submissions](https://github.com/ankidroid/Anki-Android/pulls?q=is%3Apr+merged%3A2022-02-01..2022-02-28) (I'll do my best to follow this with the same process for March in a week)

If you are interested in compensation for this work, the process with details is here:

https://github.com/ankidroid/Anki-Android/wiki/OpenCollective-Payment-Process#how-to-get-paid

(I only post one comment per person to avoid spamming you, regardless of the number of PRs merged, but this note applies to all PRs merged for in the month of January)

Thanks!

**Pull request merged**